Shared.Tests/ApiTest.cs

```diff
@@ -1348,5 +1348,25 @@ public void TestSignParameters()
 
             StringAssert.AreNotEqualIgnoringCase(m_api.SignParameters(paramsSetOne), m_api.SignParameters(paramsSetTwo), "The signatures are equal.");
         }
+
+        [Test]
+        public void TestClientHints()
+        {
+            const string assertMessage = "Should not implement responsive behaviour if client hints is true.";
+            m_api.ClientHints = true;
+
+            Transformation trans = new Transformation()
+                   .Crop("scale")
+                   .Width("auto")
+                   .Dpr("auto");
+
+            string testTag = m_api.UrlImgUp.Transform(trans).BuildImageTag("sample.jpg");
+            m_api.ClientHints = false; //reset to default value to prevent the impact on other tests
```

Comment:
this is a really bad practice, if it might affect other tests, don't use m_api, but create a new local instance or copy m_api and use it.